spec/lib/authorities/linked_data/search_config_spec.rb

```diff
@@ -11,12 +11,12 @@
   let(:term_only_config) { Qa::Authorities::LinkedData::Config.new(:LOD_TERM_ONLY_CONFIG).search }
   let(:encoding_config) { Qa::Authorities::LinkedData::Config.new(:LOD_ENCODING_CONFIG).search }
 
-  let(:ldpath_results_config) do
+  let(:predicate_results_config) do
     {
```

Comment:
ldpath_results_config is no longer needed because lod_full_config fixture now defines them.  The predicate definitions were removed from the fixture, so they are defined here for the predicate tests to be able to continue to pass.  They will eventually be removed when support for predicates is removed from the config processing classes.